### DISCUSSION TITLE
proteomics MS imaging data processing

### DISCUSSION THREAD

**Discussion by yufree** [MannLabs/alphabase#147] [General]
Hi, Great works for proteomics! 
I wonder if this ecosystem will cover MS imaging data with/without ion mobility in the future. Current open source solution is [HiTMaP](https://www.nature.com/articles/s41467-021-23461-w) while it will not process data with ion mobility for annotation purpose.

**Comment by jalew188** [comment]
Hi @yufree , if the data format is similar, I think we can support that

**Reply by yufree** [reply]
Thanks. For MSI, the open source format is imzML, which is similar to mzML with additional spatial information(x,y,z of the frame). There is a python package [pIMZ](https://github.com/mjoppich/pIMZ) to perform the MSI segmentation analysis. However, no annotation tool is available for proteomics MSI data. It might be worthwhile to use HDF5 to store MSI data as the size of high spatial resolution data are around 50GB pre sample with ion mobility.

**Reply by jalew188** [reply]
Is it similar to timsTOF data?

**Reply by yufree** [reply]
Yes. For Bruker, .D file will contain coords if they are collected in imaging mode.

**Reply by jalew188** [reply]
Maybe check our alphatims repo for timsTOF loader: https://github.com/MannLabs/alphatims
